README.rst

```diff
@@ -83,6 +83,45 @@ Gaia will automatically detect the folder of the binary and will place all data
 Usage
 -----
 
+Run-time Arguments
+~~~~~~~~~~~~~~~~~~
+
+It is possible to define run-time arguments in three ways.
+
+1. As command-line arguments
+
+For example:
+
+.. code:: bash
+    ./cmd/gaia/main -homepath=${PWD}/tmp -dev=true
+
+2. Environment Properties
+
+Environment variables can be used to overwrite any configuration settings / defaults that are already defined in Gaia.
+
+All Environment variables must be pre-fixed with `GAIA_` in order to avoid collision with other variables.
+
+For example:
+
+.. code:: bash
```

Comment:
@michelvocks How do I format code? :O This isn't showing up. :O

Reply:
You need to add an empty line after `.. code:: bash` for example:

.. code: bash

    #!/bin/bash
    echo hello world

Reply:
Ah, thanks 🙇 :)